puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_test_pull_request.yaml

```diff
@@ -0,0 +1,54 @@
+- job:
+    name: packaging_test_pull_request
+    logrotate:
+      daysToKeep: 45
+      numToKeep: -1
+    concurrent: true
+    properties:
+      - github:
+          url: https://github.com/theforeman/foreman-packaging
+    parameters:
+      - string:
+          name: branch
+          default:
+          description: "Base branch to merge PR into"
+      - pr_parameters
+    scm:
+      - foreman-rpm-packaging-pr
+    triggers:
+      - github_pr:
+          context: 'foreman-packaging'
```

Comment:
Maybe context here would be better as a scratch-build or test-build? or package? something to reflect that its doing a test of scratch build

Reply:
For Debian it ends up on https://stagingdeb.theforeman.org/ and doesn't know the term scratch. Just packaging?

Reply:
Sure - packaging works for now. In the future I envision splitting deb from rpm to make this clearer on a given PR.

Reply:
ACK pending this little update

Reply:
updated